### ISSUE TITLE
Changes required to support more detailed firmware information for LC Big Data boxes 

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#3420]
Support display of PNOR side for S822LC for Big Data boxes 
These are boxes using Supermicro Firmware. 

```
[root@stratton01 ~]# rinv briggs01 firm
briggs01: BMC Firmware: 1.21
```

Only the BMC firmware is displayed, but PNOR side is not.

**Comment by whowutwut** [comment]
Assigned to @zet809  for now, will also take a look if I have some cycles

**Comment by gurevichmark** [comment]
I think this is part of Adriana's firmware update work. 
Once her code for firmware update is finished, both Host and BMC firmware levels should be returned.

**Comment by whowutwut** [comment]
@gurevichmark No, it's on IPMI managed boxes running Supermicro's firmware, not AMI or OpenBMC.

**Comment by gurevichmark** [comment]
@zet809 I think another pull request is needed for this issue - to fix the `rflash -c` output.
Currently it returns:
```
[root@c910f02c05p03 p9]# rflash boston02 -c
boston02: Node firmware version for SMC BMC FW   component:   0.00 00000000
boston02: Node firmware version for PNOR FW      component:   0.00 00000000
boston02: Node firmware version for  component:   0.00 00000000
[root@c910f02c05p03 p9]#
```

**Comment by whowutwut** [comment]
@zet809 can you comment on Marks question above?

**Comment by zet809** [comment]
hi, @gurevichmark , seems the checking logic miss something.
It is null before
```
[root@c910f02c05p03 xcat]# rflash stratton01 -c
stratton01: Node firmware version for SMC BMC FW   component:   0.00 00000000
stratton01: Node firmware version for PNOR FW      component:   0.00 00000000
stratton01: Node firmware version for  component:   0.00 00000000
``` 
But, after run ipmitool hpm check, the rflash works:
```
[root@c910f02c05p03 xCAT_plugin]# rflash stratton01 -c
stratton01: Node firmware version for SMC BMC FW   component:   1.14 00000000
stratton01: Node firmware version for PNOR FW      component:   9.20 20160000
stratton01: Node firmware version for  component:   0.00 00000000
```
